### PR TITLE
Fix docker files for the development

### PR DESCRIPTION
**Pull request by tomijarvi**
Alpine caused several problems in the docker build so this has been turned to use helsinkitest images.

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -1,35 +1,36 @@
 # Pull base image
-FROM python:3.6-alpine
+FROM helsinkitest/python-node:3.6-8-slim
 
 # Set environment varibles
 ENV PYTHONDONTWRITEBYTECODE 1
 ENV PYTHONUNBUFFERED 1
```
Comment (frwickst) [inline Dockerfile]: Not needed as the base image already defines this

File: Dockerfile [hunk 2]
```diff
@@ -1,35 +1,36 @@
...
-# Install dependencies
+# Install packages
 COPY requirements.txt .
-RUN apk add --no-cache --virtual .build-deps build-base git libxslt-dev zlib-dev postgresql-dev \
-    && git config --global http.postBuffer 524288000 \
-    && pip install --upgrade pip \
-    && pip install --no-cache-dir -r requirements.txt --src /usr/local/src \
-    && apk add --virtual .runtime-deps postgresql-client \
-    && apk del .build-deps
+RUN apt-get update \
```
Comment (frwickst) [inline Dockerfile]: While this is ok and works as expected, if you want to write less there is a helper function in the image as well :)
```
RUN apt-install.sh \
    build-essential \
    gettext \
    libjpeg-dev \
    libpq-dev \
    netcat \
&& pip install --no-cache-dir -r requirements.txt --src /usr/local/src \
&& apt-cleanup.sh build-essential
```

Comment (tomijarvi) [reply Dockerfile]: Hmm.. nice, that is a smart move to use scripts and if ever needed, there can add also additional packages. Don't know yet what for but it's possible 😄 This is now changed to use scripts.

File: Dockerfile (hunk 1 above)
Comment (tomijarvi) [reply Dockerfile]: This is removed.

**Pull request merged**